devel/python/python/ert_gui/models/mixins/connectorless/relative_weights_model.py

```diff
@@ -0,0 +1,17 @@
+from ert_gui.models import ErtConnector
```

Comment:
Since you are not doing any normalization or anything else then I assume you can use a default implementation of this